content/docs/fairing/gcp-kubeflow-notebook.md

```diff
@@ -0,0 +1,47 @@
++++
+title = "Train and Deploy on GCP from a Kubeflow Notebook"
+description = "Use Kubeflow Fairing to train and deploy a model on Google Cloud Platform (GCP) from a notebook that is hosted on Kubeflow."
+weight = 35
++++
+
+This guide introduces you to using Kubeflow Fairing to train and deploy a
```

Comment:
Since we currently don't have an intro to Fairing in the docs (issue https://github.com/kubeflow/website/issues/611) how about linking to the Fairing repo on the words "Kubeflow Fairing", to give readers a clue about what Fairing is.

Reply:
Sounds good, added